firestore-emulator/typescript-quickstart/package.json

```diff
@@ -3,22 +3,19 @@
   "version": "1.0.1",
   "description": "Cloud Firestore emulator testing, with TypeScript",
   "scripts": {
-    "pretest": "eslint test/*.ts && tsc",
+    "format": "prettier --write **/*.ts",
+    "pretest": "tsc",
     "test": "mocha"
   },
   "devDependencies": {
-    "@firebase/testing": "0.3.0",
+    "@firebase/app": "^0.3.5",
+    "@firebase/testing": "0.3.3",
     "@types/mocha": "5.2.5",
-    "eslint": "5.7.0",
```

Comment:
Suggest adding tslint in place of the now-missing eslint.

Reply:
I removed `eslint` because I found it overly prescriptive about minor things like javadocs on tests, which I found unhelpful when trying to set up a quickstart. I feel like this may be a configuration issue, but I don't know much about these tools.